app/concepts/todo/create.rb

```diff
@@ -0,0 +1,9 @@
+class Todo::Create < Trailblazer::Operation
+	def process(params)
+		title = params[:title] || ''
+		description = params[:description] || ''
+		user = User.find(params[:user_id])
+		@valid = user.todos.create({title: title, 
+			description: description})
+	end
+end
```

Comment:
Honestly, doing this  was a good learning experience. Got a bit tense while doing this thou. I was a bit stuck here at first and its getting late. I just did what I could.

Sure, I'll give it a go.